hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
Ok - this makes me worry about some related things... the x and y scales in metadata, should (rightly) record the step size in x and y, which may be asymmetrical (although that's not necessarily very likely). More importantly it may have little to do with the probe size, which if I understand correctly is what the analysis really needs - so there's currently an assumption that the probe size is matched to the step size?

If that's the case - I think we need to make very clear in the documentation at least that if that's not the case the results will not mean much.